packages/client/src/app/pages/dashboard/approval-queue/approval-queue.component.html

```diff
@@ -54,14 +54,14 @@ <h4 title="Last Updated"><i-feather name="calendar"></i-feather>{{ entry.workToA
                         </div>
                     </div>
                     <div class="claimed-by" *ngIf="checkIfClaimed(entry) || checkIfClaimedByThisUser(entry)">
-                        <img class="claimed-avatar" src="{{ entry.claimedBy.profile.avatar }}">
+                        <img matTooltip="Claimed by {{entry.claimedBy.username | toLocaleString}}" matTooltipClass="offprint-tooltip" class="claimed-avatar" src="{{ entry.claimedBy.profile.avatar }}">
```

Comment:
I'd recommend against doing a `toLocaleString` here--Usernames probably shouldn't be modified to the whims of the user's browser locale.